pkg/frontend/mysql_cmd_executor.go

```diff
@@ -149,23 +149,28 @@ func (mce *MysqlCmdExecutor) GetRoutineManager() *RoutineManager {
 }
 
 func (mce *MysqlCmdExecutor) RecordStatement(ctx context.Context, ses *Session, proc *process.Process, sql string, beginIns time.Time) context.Context {
-	statementId := util.Fastrand64()
 	sessInfo := proc.SessionInfo
-	txnID := uint64(0)
+	// HEAD
+	var stmID [16]byte
+	binary.BigEndian.PutUint64(stmID[:8], util.Fastrand64())
```

Comment:
Use a real uuid generator function.

Reply:
If we are smart and careful, we can encode some information in uuid.   Maybe version 1 is good enough.

This is extremely useful in practice, because you can get uuid of the statement, then figure out the time the query is run, and you can get logs of say, +/- 1 hour of the query, instead of scanning whole database for the statement.

Reply:
good to know.
will discuss with @daviszhen 
while here is make sure statement_id/session_id can save uuid content.